charts/trino/values.yaml

```diff
@@ -447,6 +447,15 @@ secretMounts: []
 # ```
 
 coordinator:
+  deployment:
+    progressDeadlineSeconds: ~
+    revisionHistoryLimit: ~
+    strategy: {}
+    #  rollingUpdate:
```

Comment:
Put this into a code block, and annotate it as an example. See other properties as an example how it should look like